src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -35,10 +37,19 @@ private val LOG = LoggerFactory.getLogger("CosmasGoogleCloudService")
  *
  * @author Aleksandr Fedotov (iisuslik43)
  */
-class CosmasGoogleCloudService(private val bucketName: String,
-                               private val storage: Storage = StorageOptions.getDefaultInstance().service) : CosmasGrpc.CosmasImplBase() {
+class CosmasGoogleCloudService(private val freeBucketName: String, private val paidBucketName: String,
```

Comment:
Put a linebreak after the first argument.

Reply:
done